src/styles/main.scss

```
@@ -1,3 +1,77 @@
+$cell-size: 100px;
+$font-size: 30px;
+$gap-size: 1px;
+$days:
+  "mon" 1,
+  "tue" 2,
+  "wed" 3,
+  "thu" 4,
+  "fri" 5,
+  "sat" 6,
+  "sun" 7
+;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+}
+
 body {
   margin: 0;
 }
+
+.page {
+  display: flex;
+  justify-content: center;
+
+  margin: 100px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $gap-size;
+  width: (7 * $cell-size) +($gap-size * 6);
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid black;
+    background-color: #eee;
+  }
+
+  @for $index from 1 through 31 {
+    &__day:nth-child(#{$index}):before {
+      content: "#{$index}";
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} > :nth-child(n +#{$i + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $count in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($cell-size + $gap-size) * $count;
+    }
+  }
+
+  &__day:hover {
+    background-color: pink;
+    transform: translateY(-20px);
+    transition: all 0.5s;
```

transition надо указывать на __day по умолчанию, потому что таким образом анимация работает только в одну сторону(когда наводишь на клеточку), но не в другую(когда убираешь мышку с клеточки)
```suggestion
    transition: transform 0.5s;
```
Указывай какое свойство ты анимируешь, all - bad practice